site/content/faq/what-is-zaps-assurance-case.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "What is ZAP's assurance case?"
+type: faq
+category: Detailed Questions
+weight: 7
+---
+
+## Basic Threat Model & Trust Boundaries
+
+ZAP is a Java application which is meant to be used primarily in Single user and CI/CD deployment models. As such protection of ZAP and it's associated/generated data is outside the scope of ZAP's sphere of influence (ex: if a system which houses ZAP can be physically stolen, then all the data on the system is at risk).
+
+While attempts are made to ensure that third parties cannot adversely impact users of ZAP, users should always be aware of what their targets are and what data or services ZAP may expose. To that end:
+
+- Users identify which "targets" they interact with.
+- The Web API and proxy are available only locally by default.
+- Access to the Web API requires the use of a key/token and identified permitted clients by default.
+- Scripts and add-ons are added based on user selection/choice.
+	- Scripts and add-ons which are maintained by the core team or advertised via the ZAP Marketplace are reviewed prior to release, to ensure at the very least that they do not do anything obviously malicious.
```

Comment:
Is the indentation on purpose?

Reply:
Yes, but I guess it doesn’t have to be 🤷‍♂️